### PR TITLE
Update UserGuide for new command

### PR DESCRIPTION
**Pull request by dhlee03**
# Changes for UserGuide
-  update Add command
- update Edit command
- add View command
- add Link command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -93,19 +95,19 @@ Format: `help`
 
 Adds a person to HackLink.
 
-Format: `add n/<name>, e/<email>, p/<phone>, a/<address>, c/<category>`
+Format: `add nn/NAME p/PHONE e/EMAIL c/CATEGORY g/GROUP`
```
Comment (meenulekha) [inline docs/UserGuide.md:98]: Edit to n/NAME

**Review by meenulekha — CHANGES_REQUESTED** [review]
LGTM

File: docs/UserGuide.md [hunk 2]
```diff
@@ -93,19 +95,19 @@ Format: `help`
 
 Adds a person to HackLink.
 
-Format: `add n/<name>, e/<email>, p/<phone>, a/<address>, c/<category>`
+Format: `add n/NAME p/PHONE e/EMAIL c/CATEGORY g/GROUP`
```
Comment (bachletuankhai) [inline docs/UserGuide.md:98]: GROUP should be optional? (inside [])

**Review by bachletuankhai — APPROVED** [review]
LGTM

**Review by meenulekha — APPROVED** [review]
LGTM

**Pull request merged**